bundle/src/main/java/com/adobe/acs/commons/dam/impl/ProjectTaskAssetMoverHandler.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2016 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package com.adobe.acs.commons.dam.impl;
+
+import com.day.cq.search.QueryBuilder;
+
+import org.apache.felix.scr.annotations.Activate;
+import org.apache.felix.scr.annotations.Component;
+import org.apache.felix.scr.annotations.ConfigurationPolicy;
+import org.apache.felix.scr.annotations.Properties;
+import org.apache.felix.scr.annotations.Property;
+import org.apache.felix.scr.annotations.PropertyOption;
+import org.apache.felix.scr.annotations.Reference;
+import org.apache.felix.scr.annotations.Service;
+import org.apache.sling.api.resource.ResourceResolverFactory;
+import org.apache.sling.commons.osgi.PropertiesUtil;
+import org.apache.sling.commons.scheduler.Scheduler;
+import org.osgi.service.event.EventConstants;
+
+import java.util.Map;
+
+
+@Component(
+        label = "ACS AEM Commons - Project Task Move Handler",
+        description = "Create an OSGi configuration to enable this feature.",
+        metatype = true,
+        immediate = true,
+		configurationFactory = true,
+        policy = ConfigurationPolicy.REQUIRE
+)
+@Properties({
+        @Property(
+                label = "Event Topics",
+                value = {"com/adobe/granite/taskmanagement/event"},
+                description = "[Required] Event Topics this event handler will to respond to. Defaults to: com/adobe/granite/taskmanagement/event",
+                name = EventConstants.EVENT_TOPIC,
+                propertyPrivate = true
+        ),
+
+        /* Event filters support LDAP filter syntax and have access to event.getProperty(..) values */
+        /* LDAP Query syntax: https://goo.gl/MCX2or */
+        @Property(
+                label = "Event Filters",
+                // Only listen on events associated with nodes that end with /jcr:content
+                value = "(&(TaskTypeName=dam:review)(EventType=TASK_COMPLETED))",
+                description = "Event Filters used to further restrict this event handler; Uses LDAP expression against event properties. Defaults to: (&(TaskTypeName=dam:review)(EventType=TASK_COMPLETED))",
+                name = EventConstants.EVENT_FILTER,
+                propertyPrivate = false
+        )
+})
+@Service
+public class ProjectTaskAssetMoverHandler extends ReviewTaskAssetMoverHandler {
+    
+    private static final String DEFAULT_APPROVED_TERM = "approved";
```

Comment:
For my own edification; what is the significance of this value? Will this simply show up for display? Can it be anything without impact to anything in projects (for example, Asset Review Tasks has very specific values required)